app/experimenter/experiments/constants/nimbus.py

```diff
@@ -0,0 +1,106 @@
+class NimbusConstants(object):
+    # Status stuff
+    STATUS_DRAFT = "Draft"
+    STATUS_REVIEW = "Review"
+    STATUS_SHIP = "Ship"
+    STATUS_ACCEPTED = "Accepted"
+    STATUS_LIVE = "Live"
+    STATUS_COMPLETE = "Complete"
+    STATUS_REJECTED = "Rejected"
+
+    STATUS_DRAFT_LABEL = "Draft"
+    STATUS_REVIEW_LABEL = "Ready for Sign-Off"
+    STATUS_SHIP_LABEL = "Ready to Ship"
+    STATUS_ACCEPTED_LABEL = "Accepted by Normandy"
+    STATUS_LIVE_LABEL = "Live"
+    STATUS_COMPLETE_LABEL = "Complete"
+    STATUS_REJECTED_LABEL = "Rejected"
+
+    STATUS_CHOICES = (
+        (STATUS_DRAFT, STATUS_DRAFT_LABEL),
+        (STATUS_REVIEW, STATUS_REVIEW_LABEL),
+        (STATUS_SHIP, STATUS_SHIP_LABEL),
+        (STATUS_ACCEPTED, STATUS_ACCEPTED_LABEL),
+        (STATUS_LIVE, STATUS_LIVE_LABEL),
+        (STATUS_COMPLETE, STATUS_COMPLETE_LABEL),
+        (STATUS_REJECTED, STATUS_REJECTED_LABEL),
+    )
+
+    MAX_DURATION = 1000
+
+    # Version stuff
+    VERSION_CHOICES = (
```

Comment:
I haven't seen exactly how we use these, but would we get clearer code elsewhere when using this if this was a list of namedtuple's instead of a tuple of tuples where our access is going to be by index only?

Reply:
Oh that's a good point this might be a good opportunity to use enums!

Reply:
An enum would also help with readability.

Reply:
The version choice might be better as a namedtuple.

Reply:
Oh django has a dealy for this now https://docs.djangoproject.com/en/3.1/ref/models/fields/#enumeration-types